docker-tox/setup.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+PYTHON_VERSIONS=(2.7.10 3.5.2)
+PYTHON_GLOBAL_VERSION=2.7.10
+PYTHON_PIP_VERSION=7.1.2
+
+# Install pyenv
+curl -L https://raw.githubusercontent.com/yyuu/pyenv-installer/master/bin/pyenv-installer -o /pyenv-installer
```

Comment:
Feels dangerous to install things out of the internet that can actually be a moving target.